src/services/apiService.ts

```diff
@@ -0,0 +1,33 @@
+import { logger } from "@blockr/blockr-logger";
+import { Transaction } from "@blockr/blockr-models";
+import Axios from "axios";
+
+const publicApiUrl = "https://public.blockr.verux.nl";
```

Comment:
Please make these constants outside the scope of a class with upper case, so `PUBLIC_API_URL`.